src/library/scala/collection/View.scala

```diff
@@ -143,7 +143,7 @@ object View extends IterableFactory[View] {
   }
 
   /** A view that partitions an underlying collection into two views */
-  @SerialVersionUID(3L)
+  //@SerialVersionUID(3L)
```

Comment:
I think the bug is the other way here, so we could leave this for now. https://github.com/scala/bug/issues/10945

Reply:
As I suspected, thx.